scalardb/project.clj

```diff
@@ -10,9 +10,10 @@
                  [cassandra "0.1.0-SNAPSHOT"]
                  [cc.qbits/alia "4.3.6"]
                  [cc.qbits/hayt "4.1.0"]]
+  :repositories {"sonartype" "https://oss.sonatype.org/content/repositories/snapshots/"}
   :profiles {:dev {:dependencies [[tortue/spy "2.0.0"]]
                    :plugins [[lein-cloverage "1.1.2"]]}
-             :use-released {:dependencies [[com.scalar-labs/scalardb "3.7.0"
+             :use-released {:dependencies [[com.scalar-labs/scalardb "4.0.0-SNAPSHOT"
```

Comment:
It's a comment for the `user-jars`.
Do you think you want to keep `use-jars`?
I feel we can delete it since it's bothersome to maintain the packages.